apps/marketplace/components/Opportunity/Opportunity.js

```diff
@@ -74,6 +74,18 @@ const getBriefCategory = (domains, briefCategory) => {
   return category ? category.name : null
 }
 
+const getQuestionsCloseDate = brief => {
+  if (brief.dates.questions_close) {
+    return new Date(brief.dates.questions_close)
+  }
+
+  if (getClosingTime(brief)) {
+    return getBriefLastQuestionDate(new Date(getClosingTime(brief)))
+  }
+
+  return false
```

Comment:
keep return types consistent..
```suggestion
  return null
```

Reply:
Should we return `null` for `showATMObjectives`?